app/api/endpoints/chat.py

```diff
@@ -21,8 +21,11 @@ async def send_message(
     template_name: str = Query(
         "chat_default",
         description="The template name without suffix, e.g. chat_default, default to 'chat_default' for gchat"),
```

Comment:
could you add a the link https://github.com/waynesun09/notify-service/blob/main/app/templates/build/chat_default.jinja to 'chat_default'?

Reply:
sure